src/main/frontend/handler/plugin_config.cljs

```diff
@@ -0,0 +1,130 @@
+(ns frontend.handler.plugin-config
+  "This system component encapsulates the global plugin.edn and depends on the
+  global-config component. This component is only enabled? if both the
+  global-config and plugin components are enabled. plugin.edn is automatically updated
+when a plugin is installed, updated or removed"
+  (:require [frontend.handler.global-config :as global-config-handler]
+            ["path" :as path]
+            [promesa.core :as p]
+            [borkdude.rewrite-edn :as rewrite]
+            [frontend.fs :as fs]
+            [frontend.state :as state]
+            [frontend.handler.notification :as notification]
+            [frontend.handler.common.plugin :as plugin-common-handler]
+            [clojure.edn :as edn]
+            [clojure.set :as set]
+            [clojure.pprint :as pprint]
+            [malli.core :as m]
+            [malli.error :as me]
+            [frontend.schema.handler.plugin-config :as plugin-config-schema]
+            [cljs-bean.core :as bean]
+            [lambdaisland.glogi :as log]))
+
+(defn plugin-config-path
+  "Full path to plugins.edn"
+  []
+  (path/join (global-config-handler/global-config-dir) "plugins.edn"))
+
+(def common-plugin-keys
+  "Vec of plugin keys to store in plugins.edn and to compare with installed-plugins state"
+  (->> plugin-config-schema/Plugin rest (mapv first)))
```

Comment:
Malli's schema is just data and allows for easy downstream usages like this. This is more difficult to do with spec

Reply:
Love the data-oriented design of Malli, just curious that what's your experience with both Malli and Spec? Anything special that you think Spec is doing a better job than Malli?

Reply:
I've used spec more, especially with its generators. I don't have much experience with malli yet but so far it's promising - mostly feature parity with spec, new features continue to be released and some features are more practical e.g. transformations and json schema support. Spec is probably more readable as it doesn't encourage inline definitions like malli does.  Spec may have better integration with test.check. https://github.com/clojure/spec-alpha2 is supposed to address the shortcomings of spec but there has been no commitment on release for a couple years so I don't think it's a good idea to use it